packages/serverpod_test/lib/src/database_proxy.dart

```diff
@@ -0,0 +1,234 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test/src/transaction_manager.dart';
+
+import 'with_serverpod.dart';
+
+/// A database proxy that forwards all calls to the provided database.
+class TestDatabaseProxy implements Database {
```

Comment:
Suggestion: Rename file to match the class in the file.

Reply:
Ah shoot, thanks for catching!